app/views/candidates/schools/_phase2.html.erb

```diff
@@ -52,116 +39,156 @@
+        </strong>
+      </p>
 
-    <div class="govuk-body">
-      <%= format_school_availability @school %>
+      <ul id="school-websites">
+        <%- if @presenter.website.present? -%>
+        <li id="school-website">
+          <%= link_to "#{@school.name} website", @presenter.website %>
+        </li>
+        <%- end -%>
+
+        <li>
+          <%= link_to "#{@school.name} Get Information About Schools",
+                      gias_school_url(@presenter.urn) %>
+        </li>
+
+        <li>
+          <%= link_to "Ofsted report: #{@school.name}",
+                      ofsted_report_url(@presenter.urn) %>
+        </li>
+
+        <li>
+          <%= link_to "Performance information: #{@school.name}",
+                      performance_report_url(@presenter.urn) %>
+        </li>
+
+        <%- if @presenter.teacher_training_url.present? -%>
+        <li>
+          <%= link_to "Teacher training: #{@presenter.name}",
+            cleanup_school_url(@presenter.teacher_training_url) %>
+        </li>
+        <%- end -%>
+      </ul>
     </div>
 
-    <h3 class="govuk-heading-s">
-      DBS check required
-    </h3>
-
-    <p class="govuk-body">
-      <%= @school.dbs ? 'Yes' : 'No' %>
-    </p>
-
-    <h3 class="govuk-heading-s">
-      Fee
-    </h3>
-
-    <p class="govuk-body">
-      <%= @school.fee %>
-    </p>
-
-    <h3 class="govuk-heading-s">
-      Dress code
-    </h3>
-
-    <p class="govuk-body">
-      <%= @school.dress_code %>
-    </p>
-
-    <h3 class="govuk-heading-s">
-      Start and finish times
-    </h3>
-
-    <p class="govuk-body">
-      <%= @school.times %>
-    </p>
-
-    <h3 class="govuk-heading-s">
-      Parking
-    </h3>
-
-    <p class="govuk-body">
-      <%= @school.parking %>
-    </p>
-
-    <h3 class="govuk-heading-s">
-      Accessibility details
-    </h3>
-
-    <p class="govuk-body">
-      <%= @school.accessibility %>
+    <p>
+      <%= link_to 'Back to Search Results', :back,
+                  data: {controller: 'back-link'} %>
     </p>
+  </div>
 
-    <h3 class="govuk-heading-s">
-      Teacher training offered
-    </h3>
-
-    <p class="govuk-body">
-      <%= @school.training_offered %>
-    </p>
+  <div class="candidate-profile-sidebar govuk-grid-column-one-third column-top-border">
+    <dl class="govuk-summary-list govuk-summary-list--no-border vertical">
+      <%= dlist_item 'Address', id: 'school-address' do %>
+        <%= format_school_address @presenter.school, tag(:br) %>
+      <% end %>
+
+      <div id="school-availability-info" class="govuk-summary-list__row">
+        <%- if @presenter.availability_preference_fixed? -%>
+          <%= render partial: 'placement_dates', locals: { available_dates: @available_dates } %>
+        <%- else -%>
+          <dt class="govuk-summary-list__key">
+            Placement availability
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= format_school_availability(@school) %>
```

Comment:
This method calls `availability_info` on the school, I think it needs to call it on the bookings/profile